### PR TITLE
Merge latest changes into master

### PR DESCRIPTION
**Pull request by geleeroyale**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -105,9 +105,18 @@ module.exports = {
...
-    'gatsby-plugin-use-query-params'
+    'gatsby-plugin-use-query-params',
     // this (optional) plugin enables Progressive Web App + Offline functionality
     // To learn more, visit: https://gatsby.dev/offline
     // `gatsby-plugin-offline`,
+    {
+      resolve: '@sentry/gatsby',
+      options: {
+        dsn:
+          'https://de31cb89ac0045afbb9d28322cc9c040@o510515.ingest.sentry.io/5606310',
```
Comment (geleeroyale) [inline gatsby-config.js:116]: should we make this an env variable?

**Review by geleeroyale — COMMENTED** [review]
💯

File: gatsby-config.js (hunk 1 above)
Comment (geleeroyale) [reply gatsby-config.js:116]: @jamespfarrell

File: gatsby-node.js [hunk 2]
```diff
@@ -86,3 +86,22 @@ exports.onCreateNode = ({ node }) => {
...
   }
 }
+
+exports.onCreateWebpackConfig = ({ stage, loaders, actions }) => {
+  if (stage === 'build-html') {
+    actions.setWebpackConfig({
+      module: {
+        rules: [
+          {
+            test: /\@toruslabs\/torus-embed/,
```
Comment (aminlatifi) [inline gatsby-node.js:96]: @jamespfarrell Nice job 👏

**Pull request merged**